src/main/java/frc/robot/Subsystems.java

```diff
@@ -20,6 +21,7 @@ public static class SubsystemConstants {
 
     public static final boolean ELEVATOR_ENABLED = true;
     public static final boolean ARMPIVOT_ENABLED = true;
+    public static final boolean CLIMBPIVOT_ENABLED = true;
```

Comment:
Until the climb pivot is actually installed on the robot, please set this to `false`